about/people.yml

```diff
@@ -430,6 +430,17 @@ data:
     brown_directory_uuid: 78e02962-a8af-46ec-8636-61237d83cb41
     bio: Michael is a data science intern in the High Performance Computing division at CCV. He is in the Class of 2026 at Brown University studying Computer Science. His academic interests include software development, deep learning, and East Asian studies. Outside of work, you can find him playing basketball or listening to music.
     image: michael1_main.jpg
+<<<<<<< HEAD
+    name: Sima Raha
+    type: Part Time
+    team: Advanced Research Computing
+    subteam: Graphics, Software, and Data Core
+    title: Software Sustanibaility Intern 
+    github_username: SimaRaha
+    brown_directory_uuid: cf68285e-47c5-4a65-897e-187c94e4f36b
+    bio: Sima Raha is a Software Sustainability Intern at CCV. She is Class of 2025, studying Economics and Data Fluency at Brown University. Her career interests include quantitative analysis, business intelligence (BI) development, and statistical programming. She enjoys hiking, nature photography, playing frisbee, and billiards.
+    image: sima_main.jpg
+=======
```

Comment:
Looks like there's still a merge conflict marker here on line 433 and 443. The `<<<<< HEAD` line means "the following changes come from the current branch, until the `=====` line." You need to delete lines 433 and 443 in order to resolve the merge conflict.

Reply:
Oh, and line 472.

Reply:
@galenwinsor I already have line 433 in my local code. And the line 443 does not work for me to delete it. So, do I need to push the change to github?